phoenix-core/src/main/java/org/apache/phoenix/log/QueryLogDetailsWorkHandler.java

```diff
@@ -17,36 +17,29 @@
  */
 package org.apache.phoenix.log;
 
-import java.sql.SQLException;
-
+import com.lmax.disruptor.LifecycleAware;
+import com.lmax.disruptor.WorkHandler;
 import org.apache.hadoop.conf.Configuration;
 
-import com.lmax.disruptor.LifecycleAware;
-import com.lmax.disruptor.Sequence;
-import com.lmax.disruptor.SequenceReportingEventHandler;
+
+public class QueryLogDetailsWorkHandler implements WorkHandler<RingBufferEvent>, LifecycleAware {
```

Comment:
Trying to understand the change of type hierarchy. Seems like we didn't actually need the Sequence part of disruptor?

Reply:
sequenceCallback was never used in the old implementation.